src/main/resources/assets/ukulib/lang/es_es.lang

```diff
@@ -0,0 +1,16 @@
+{
+  "ukulib.position": "Posición",
+  "ukulib.position.topLeft": "Superior Izquierdo",
+  "ukulib.position.topRight": "Superior Derecho",
+  "ukulib.position.bottomLeft": "Inferior Izquierdo",
+  "ukulib.position.bottomRight": "Inferior Izquierdo",
+  "ukulib.position.middle": "Por encima barra XP",
+  "ukulib.position.desc": "Haga clic en algún lugar de la pantalla para cambiar la posición del elemento.",
+  "ukulib.position.desc.move": "Pulse las §eteclas de flecha§r para ajustar con precisión la posición del elemento.",
+  "ukulib.option.invalid": "Algunos de los valores introducidos en la configuración son inválidos: %s.\ntodavía puede salir pulsando ESC, pero la configuración no se guardará.",
+  "ukulib.option.reset": "Restaurar Valores Predeterminados",
+  "ukulib.config.title": "Ukulib Configuración",
+  "ukulib.config.buttonInOptions": "Botones En Las Opciones",
+  "ukulib.config.modMenuIntegration": "Menú de Mods Integración",
```

Comment:
since modmenu is the actual name of the mod, i think it's probably better to keep it as is?

```suggestion
  "ukulib.config.modMenuIntegration": "Modmenu Integración",
```

Reply:
Jesus, yes. Make's me look bad at this...
Menú de Mods is actually what ModMenu call themselves in spanish, but if you want to change it to the mod name, that is all fine.

Reply:
oh, I didn't know! Menú de Mods is fine then :)